src/utils/CaptainConstants.ts

```diff
@@ -2,7 +2,7 @@ import fs = require('fs-extra')
 import path = require('path')
 import EnvVars from './EnvVars'
 
-const CAPTAIN_BASE_DIRECTORY = '/captain'
+const CAPTAIN_BASE_DIRECTORY = EnvVars.CAPTAIN_BASE_DIRECTORY ?? '/captain'
```

Comment:
Suggest using double pipe instead of double question mark to account for hypothetical cases where the env var is set to an empty value. 

PS: I can make the change after merge. Just leaving the comment so I don't forget!

Reply:
Makes complete sense. Will commit when I get a chance.

Reply:
Updated